server.elm

```diff
@@ -1696,6 +1994,28 @@ editionscript = """
     var linkToEdit = @(if defaultVarEdit then "link => link" else 
      """link => link && !isAbsolute(link) ? link.match(/\?/) ? link + "&edit" : link + "?edit" : link;""");
     
+    /*
+    <div>Icons made by <a href="https://www.freepik.com/" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/"                 
+    title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/"                 
+    title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
+    */
+    var undoSVG = `<svg viewBox="0 0 20 50", height = "80", width = "40", xmlns="http://www.w3.org/2000/svg">
```

Comment:
Use mkSvg("M 9.5,12.625 11.75,19.25 17.25,15.125 M 31.5,16 C 30.25,11.875 26.375,9 22,9 16.5,9 12,13.5 12,19") instead and remove the reference to Freepik